index.rst

```diff
@@ -2,6 +2,613 @@
+
+LSST Project
+   This category is only visible to LSST project members (``LSST`` group). It has been used to debrief conferences and offer frank discussions.
+
+Planned and Possible Categories
+-------------------------------
+
+Ask LSST
+   This category, sponsored by the Project Science Team, will provide the science collaborations, and the astronomy community in general, a venue to ask questions about how LSST will operate and serve their science goals and receive official answers from the project.
+   Such a Q&A venue will offer an appealing alternative to getting answers through our technical documentation or through one-on-one conversations that don't scale.
+   Technically, this category will operate similarly to the Support category.
+
+Broadcasting to mailing lists (Community Mailbot)
+-------------------------------------------------
+
+Community was intended to replace DM's mailing lists, and it has:
+conversations no longer occur on the ``dm-devel`` and ``dm-user``
+mailing lists.  However, we also recognized that these mailing lists
+have value in reliably reaching an audience which prefers e-mail.
+Thus we built the `Community Mailbot
+<https://github.com/lsst-sqre/community_mailbot>`_ to forward new
+topics in select categories to the existing DM mailing lists.  The
+forwarded email contains the text of the original topic post along
+with an unambiguous button inviting readers to participate in the
+discussion on https://community.lsst.org. Echoing forum activity to an
+e-mail gateway has been common practice since the early days of the Internet. 
+
+SQuaRE uses Mandrill, by Mailchimp, to send these emails.
+
+Project group management
+------------------------
+
+As discussed, we assign project staff to 'groups' within Community
+that offer higher Discourse trust levels and access to private
+categories. Currently this assignment is managed manually by SQuaRE
+and DM T/CAMs. As Community's use grows across the project, this may
+arrangement will scale poorly.
+
+SQuaRE is highly desirous of interfacing to the LSST Contacts via a
+standard programmatic API, which is not possible with the current
+Contacts DB implementation in order to ensure that group access in
+Community and other SQuaRE services is kept in sync with the Project's
+master list. 
+
+.. _confluence:
+
+Confluence Wiki
+===============
+
+DM uses Confluence wikis, although their role is being diminished with
+the introduction of |clo| and the |ltd| publishing paradigm (including
+Technical Notes, the new Developer Guide and software documentation).
+
+SQuaRE dissuades software documentation in wikis, since it cannot be
+managed with standard software release tools, cannot be tested by our
+continuous integration harness, is "out of sight out of mind" for the
+developers, and is hard to maintain. We are in the process of
+migrating all software documentation from Confluence to other, better
+harnesses.
```

Comment:
I've seen it being used for 1) planning spreadsheets, 2) design documents (both within DM and LSST-wide in Operations planning and systems engineering).
Some people are also using Dropbox to pass around information like presentations.